docs/content/quickstart/build-real-time-data-lake-tutorial.md

```diff
@@ -0,0 +1,257 @@
+   - [flink-sql-connector-mysql-cdc-2.1-SNAPSHOT.jar](https://repo1.maven.org/maven2/com/ververica/flink-sql-connector-mysql-cdc/2.1-SNAPSHOT/flink-sql-connector-mysql-cdc-2.1-SNAPSHOT.jar)
+   - [iceberg-flink-runtime-0.12.0.jar](https://repo.maven.apache.org/maven2/org/apache/iceberg/iceberg-flink-runtime/0.12.0/iceberg-flink-runtime-0.12.0.jar)
+   - [flink-shaded-hadoop-2-uber-2.7.5-10.0.jar](https://repo.maven.apache.org/maven2/org/apache/flink/flink-shaded-hadoop-2-uber/2.7.5-10.0/flink-shaded-hadoop-2-uber-2.7.5-10.0.jar)
+2. This tutorial will use the local filesystem as the backend of Iceberg. To make it can be accessed by the containers, 
+   we mount the volume `/Users/yuxia/demo/iceberg` in the local filesystem into `/home/iceberg` for the containers in this `docker-compose.yml` file.
+   You can change `/Users/yuxia/demo/iceberg` to any other directory you like in your local file system.
+3. All the following commands for entering the container should be executed in the directory of the `docker-compose.yml` file.
+
+To start all containers, run the following command in the directory that contains the `docker-compose.yml` file:
+```shell
+docker-compose up -d
+```
+This command automatically starts all the containers defined in the Docker Compose configuration in a detached mode. Run `docker ps` to check whether these containers are running properly.
+We can also visit [http://localhost:8081/](http://localhost:8081/) to see if Flink is running normally.
+![Flink UI](/_static/fig/real-time-data-lake-tutorial/flink-ui.png "Flink UI")
+
+### Preparing data in databases
+1. Enter mysql's container:
+    ```shell
+    docker-compose exec mysql mysql -uroot -p123456
+    ```
+2. Create databases/tables and populate data:
+
+   Create a logical sharding table `user` sharded in different databases and tables physically, and one of them misses a column.
+   ```sql
+    CREATE DATABASE user_db_1;
+    USE user_db_1;
+    CREATE TABLE user_table_1 (
+      id INTEGER NOT NULL PRIMARY KEY,
+      name VARCHAR(255) NOT NULL DEFAULT 'flink',
+      address VARCHAR(1024),
+      phone_number VARCHAR(512),
+      email VARCHAR(255)
+    );
+    INSERT INTO user_table_1 VALUES (11100,"user_11100","Shanghai","123567891234","user_11100@foo.com");
+   
+    CREATE TABLE user_table_2 (
+      id INTEGER NOT NULL PRIMARY KEY,
+      name VARCHAR(255) NOT NULL DEFAULT 'flink',
+      address VARCHAR(1024),
+      phone_number VARCHAR(512),
+      email VARCHAR(255)
+    );
+   INSERT INTO user_table_2 VALUES (12100,"user_12100","Shanghai","123567891234","user_12100@foo.com");
+   ```
+   ```sql
+   CREATE DATABASE user_db_2;
+   USE user_db_2;
+   CREATE TABLE user_table_1 (
+     id INTEGER NOT NULL PRIMARY KEY,
+     name VARCHAR(255) NOT NULL DEFAULT 'flink',
+     address VARCHAR(1024),
+     phone_number VARCHAR(512)
+   );
+   INSERT INTO user_table_1 VALUES (21100,"user_21100","Shanghai","123567891234");
```

Comment:
The primary key can be duplicate across different tables, this is also a good example to show the power of metadata columns. e.g. `db_1.user_1` and `db_2.user_1` both contains primary key `101`.